ozhera-log/log-manager/src/main/java/com/xiaomi/mone/log/manager/service/extension/store/DefaultStoreExtensionService.java

```diff
@@ -62,41 +64,58 @@ public class DefaultStoreExtensionService implements StoreExtensionService {
     @Resource
     private MilogLogstoreDao logStoreDao;
 
+    @Resource
+    private DorisLogStorageService dorisLogStorageService;
```

Comment:
The excuse needs to replace this in the future.

Reply:
This is a progress and we look forward to increasing it in the future

Reply:
> The excuse needs to replace this in the future.

This is a progress and we look forward to increasing it in the future